compiler-core/src/docs.rs

```diff
@@ -202,19 +239,23 @@ pub fn generate_html(
         search_indexes.push(SearchIndex {
             doc: module.name.to_string(),
             title: module.name.to_string(),
-            content: module.ast.documentation.iter().join("\n"),
+            content: documentation_content,
             url: format!("{}.html", module.name),
         });
 
+        let page_title = format!("{} · {} · v{}", name, config.name, config.version);
+        let page_meta_description = "";
```

Comment:
Here it would best if we could add an optional short meta description in `.gleam` module files @lpil?

Reply:
Should I create an issue for this?

Reply:
We could also colocate optional meta data in the file system - not sure how noisy that would get.